qemu/tests/cfg/hotplug_mem_share_discard_data.cfg

```diff
@@ -0,0 +1,35 @@
+- hotplug_mem_share_discard_data:
+    only Linux
+    type = hotplug_mem_share_discard_data
+    virt_test_type = qemu
+    login_timeout = 240
+    start_vm = no
+    not_preprocess = yes
+    prealloc_mem = yes
+    slots_mem = 4
+    maxmem_mem = 32G
+    mem_fixed = 4096
+    mem_devs = mem0
+    use_mem_mem0 = no
+    size_mem_mem0 = ${mem_fixed}M
+    guest_numa_nodes = "node0"
+    node_num = 1
+    numa_memdev_node0 = mem-mem0
+    backend_mem = memory-backend-file
+    set_node_hugepage = yes
+    mem-path = /dev/hugepages
+    target_mems = "plug1"
+    share_mem_plug1 = yes
+    size_mem_plug1 = 1G
+    mem-path_plug1 = /dev/hugepages/test_file
+    pre_command = "sync && echo 3 > /proc/sys/vm/drop_caches && echo 1 > /proc/sys/vm/compact_memory"
+    variants:
+        - backend_ram:
+            backend_mem_plug1 = memory-backend-ram
```

Comment:
@yanan-fu 
After including this point and tested it I see that for qemu-kvm<=6.1 share: True is not being displayed in the info memdev command: 

```bash
memory backend: mem0
  size:  1073741824
  merge: true
  share: true   -----> shows since qemu6.1
```
Previous block extracted from manual steps:
https://polarion.engineering.redhat.com/polarion/redirect/project/RHELVIRT/workitem?id=RHEL-118793

Reply:
May I consider then the entire case (not only backend_ram) valid if qemu >= 6.1 ?

Reply:
I am afraid not :)
From manual case,  it only mentioned `share` with `memory-backend-ram` was supported from qemu 6.1.
For `memory-backend-file`, it was support more early.

Reply:
Changed!